docs/azure_arc_jumpstart/azure_arc_k8s/pf9/pf9_bareos/_index.md

```diff
@@ -0,0 +1,208 @@
+
+  ```shell
+  az provider show -n Microsoft.Kubernetes -o table
+  az provider show -n Microsoft.KubernetesConfiguration -o table
+  az provider show -n Microsoft.ExtendedLocation -o table
+  ```
+
+* Install the Azure Arc Kubernetes CLI extensions *connectedk8s* and *k8s-configuration* .
+
+  ```shell
+  az extension add --name connectedk8s
+  az extension add --name k8s-configuration
+  ```
+
+  Run below command to verify that it is installed.
+
+  ```shell
+  az extension list -o table
+  ```
+
+* Create Azure service principal (SP)
+  To be able to complete the scenario and its related automation, Azure service principal assigned with the *Contributor* role is required.
+
+  For creating the service principal, first login to your Azure account.
+
+  ```shell
+  az login
+  ```
+
+  Login by opening the [Microsoft devicelogin page](https://microsoft.com/devicelogin) on browser and enter the code to authenticate.
+
+  An example output of successful authentication is shown below;
+
+  ![MSFT Device Login](./01.png)
+
+  Once you have a successful login, create the Azure service principal (SP).
+
+  ```shell
+  az ad sp create-for-rbac -n "<service-principal-name>" --role contributor
+  ```
+
+  See below example of creating a service principal.
+
+  ![Create Service Principal](./02.png)
+
+  > *Note : It is highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).*
+
+* Create an Azure resource group where you want the PMK cluster to show up.
+
+  ```shell
+  az group create -l <Azure Region> -n <resource group name>
+  ```
+
+  See below example of creating a resource group.
+
+  ![Create Resource Group](./03.png)
+
+## Deployment
+
+This deployment consists of 3 parts.
+
+  1. Onboarding the node to Platform9 Management Plane.
+  2. Creating a PMK Cluster.
+  3. Connect the cluster to Microsoft Azure Arc.
+
+**Onboarding the node to Platform9 Management Plane**
+
+* Login to your Management Plane.
+
+  ![PMK Management Plane Login Page](./04.png)
+
+  > **If you do not have a registered Management Plane with Platform9, you can create one easily using [PMK Free Tier deployment](https://platform9.com/managed-kubernetes/)**
+
+* Onboard the Ubuntu host to your Management Plane.
+
+  For a BareOS cluster, you will need to have the nodes registered with the PMK Management Plane on which the cluster is to be deployed. For this, first add the node.
+
+  ![Onboard a node](./05.png)
+
+  This should take you to the Node onboarding page. A **pf9ctl** utility is provided to setup the nodes and get connected with Management Plane. Follow the instructions to download and install the utility, which we will use to prepare the node and connect it with your Platform9 Management Plane.
```

Comment:
```suggestion
  This should take you to the Node onboarding page. A **pf9ctl** utility is provided to setup the nodes and get connected with Management Plane. 
  Follow the instructions to download and install the utility, which we will use to prepare the node and connect it with your Platform9 Management Plane.
```

Reply:
This looks like a potential reader failure. Can we add some screenshots here to show the e2e experience?